package.json

```diff
@@ -5,8 +5,9 @@
   "main": "index.js",
   "scripts": {
     "build": "npx tsc",
-    "start": "npm run build -- -w",
-    "test": "jest",
+    "start": "node ./dist/src/index.js",
```

Comment:
smol change perhaps we should include the build command into the start command.

That way when you run `npm run start` it automatically runs `npm run build` before starting the `dist/src/index.js` file

Reply:
actually scratch that it messes with the jenkins file